### PR TITLE
Make bigtable examples output prettier.

### PR DESCRIPTION
**Pull request by coryan**

### CODE REVIEW

File: bigtable/examples/run_examples_utils.sh [hunk 1]
```diff
@@ -45,7 +45,6 @@ function run_all_instance_admin_examples {
   echo
   echo "Run create-instance example."
   $admin ../examples/bigtable_samples_instance_admin create-instance "${project_id}" "${INSTANCE}" "${zone_id}"
-  trap '$admin ../examples/bigtable_samples_instance_admin delete-instance "${project_id}" "${INSTANCE}"' EXIT
```
Comment (mbrukman) [inline bigtable/examples/run_examples_utils.sh]: This seems unrelated to the description of this PR, why the change?

File: bigtable/examples/run_examples_utils.sh [hunk 2]
```diff
@@ -156,7 +157,6 @@ function run_all_data_examples {
 
   # Create an instance to run these examples.
   $admin ../examples/bigtable_samples_instance_admin create-instance "${project_id}" "${INSTANCE}" "${zone_id}"
-  trap '$admin ../examples/bigtable_samples_instance_admin delete-instance "${project_id}" "${INSTANCE}" "${zone_id}"' EXIT
```
Comment (mbrukman) [inline bigtable/examples/run_examples_utils.sh]: Why the removal of all `trap`s?

Comment (coryan) [reply bigtable/examples/run_examples_utils.sh]: Because they remove the more important trap that kills the emulators.  Traps do not stack :disappointed:

File: bigtable/examples/run_examples_utils.sh (hunk 1 above)
Comment (coryan) [reply bigtable/examples/run_examples_utils.sh]: I should probably separate them to a different PR, they were just breaking my tests because (see next comment)..

Comment (mbrukman) [reply bigtable/examples/run_examples_utils.sh]: SGTM, if you don't mind, that would make sense.

File: bigtable/examples/run_examples_utils.sh (hunk 2 above)
Comment (mbrukman) [reply bigtable/examples/run_examples_utils.sh]: Would it be possible to have a single function that does all global cleanups and use that? Or just create a stack of them?

E.g., see:

* https://stackoverflow.com/questions/16115144/bash-save-and-restore-trap-state-easy-way-to-manage-multiple-handlers-for-trap
* https://stackoverflow.com/a/7287873

**Comment by coryan** [comment]
PTAL.

File: ci/colors.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ "${TERM:-}" = "dumb" ]; then
+  readonly COLOR_RED=""
+  readonly COLOR_GREEN=""
+  readonly COLOR_YELLOW=""
+  readonly COLOR_RESET=""
+else
+  readonly COLOR_RED=$(tput setaf 1)
```
Comment (mbrukman) [inline ci/colors.sh]: Quote RHS, please (here and below).
